packages/nextjs/components/onchain-impact-dashboard/LeaderboardTable/index.tsx

```diff
@@ -0,0 +1,141 @@
+"use client";
+
+import { useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { SearchBar } from "../SearchBar";
+import SkeletonTable from "./SkeletonTable";
+import { ArrowDownIcon } from "@heroicons/react/24/outline";
+import { ArrowUpIcon } from "@heroicons/react/24/outline";
+import { IProjectMovement } from "~~/services/mongodb/models/projectMovement";
+import "~~/styles/leaderboard.css";
+
+interface Props {
+  projects: IProjectMovement[];
+  loading: boolean;
+  selectedMetricName: string;
+}
+
+type ColumnHeader = keyof IProjectMovement;
+
+const formatMovement = (movement: number) => {
+  if (movement >= 0) {
+    return `+ ${movement}%`;
+  }
+  return `- ${movement * -1}%`;
```

Comment:
Seems like this will show double negative? "- -11%"

Reply:
This is just to make sure the padding of the `-` symbol is the same as the `+` symbol. If the movement is positive, we prefix the number with a `+ `. If the movement is negative, we flip it to positive to remove the automatic `-` symbol, and then prefix it with our own `- ` with the correct spacing.

Reply:
That makes sense! Thanks!